.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: ruby
 rvm:
-  - 1.9.2
   - 1.9.3
+  - 2.1.5
```

Comment:
could this be:
- 1.9
- 2.1
- 2.2

It then follows the latest bugfix version